pkg/xds/envoy/clusters/v2/client_side_tls_configurer.go

```diff
@@ -5,29 +5,22 @@ import (
 	envoy_core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
 	pstruct "github.com/golang/protobuf/ptypes/struct"
 
-	"github.com/kumahq/kuma/pkg/xds/envoy/tls"
-
 	"github.com/kumahq/kuma/pkg/core/xds"
 	"github.com/kumahq/kuma/pkg/util/proto"
-	"github.com/kumahq/kuma/pkg/xds/envoy"
+	envoy_metadata "github.com/kumahq/kuma/pkg/xds/envoy/metadata/v2"
+	tls_v2 "github.com/kumahq/kuma/pkg/xds/envoy/tls/v2"
```

Comment:
I would assume that under `v2` folder you don't mix versions so `tls_v2` could just be `tls`.